cmd/cmdutil/helpers.go

```diff
@@ -83,6 +94,21 @@ func ValidateArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func ValidateIntOrPercantageValue(value string) bool {
+	valid := false
```

Comment:
You can change the return value to `(valid bool)` and remove this declaration, since the nil value of a bool is false.